src/bot/routes/rooms.rs

```diff
@@ -17,11 +17,15 @@ use reqwest::StatusCode;
 use ruma::api::client::state::get_state_events_for_key;
 use ruma::events::macros::EventContent;
 use ruma::events::room::avatar::RoomAvatarEventContent;
-use ruma::OwnedMxcUri;
+use ruma::{OwnedMxcUri, RoomAliasId};
 use serde::{Deserialize, Serialize};
 use crate::domain::{CommunityData, CommunityMatrixId, Uri};
 use crate::utils::{error_chain_fmt, mxc_to_download_uri};
 
+static SERVER: &str = "virto.community";
```

Comment:
instead of `static` variables that can be unsafely mutated you probably want to use `const`s.